### PR TITLE
Linking cards to backend

### PR DESCRIPTION
**Pull request by simonarcher99**

### CODE REVIEW

File: src/homepage/Homepage.tsx [hunk 1]
```diff
@@ -1,71 +1,123 @@
...
+    },
 })
 
 export const fontTheme = createMuiTheme({
     typography: {
-        fontFamily: [
-            "Source Sans Pro",
-            "sans-serif",
-        ].join(","),
+        fontFamily: ["Source Sans Pro", "sans-serif"].join(","),
```
Comment (alicolver) [inline src/homepage/Homepage.tsx:29]: this seems silly

Comment (simonarcher99) [reply src/homepage/Homepage.tsx:29]: aww fuck knows what it's doing

File: src/homepage/Homepage.tsx [hunk 2]
```diff
@@ -1,71 +1,123 @@
...
-                return
-            }
-            return res.json()
-        }).then(res => {
-            console.log(res.data)
-            setMatchData(res.data)
+                Authorization: getJWT(),
+            },
+        })
+            .then((res) => {
```
Comment (alicolver) [inline src/homepage/Homepage.tsx]: this formatting seems off

File: src/leaderboard/PointCard.tsx [hunk 3]
```diff
@@ -1,57 +1,109 @@
...
+        display: "flex",
         justifyContent: "space-between",
         alignItems: "center",
         color: "#9a0c34",
         "& > div": {
             width: "25%",
             height: "75px",
-            boxShadow: "rgba(0, 0, 0, 0.3) 0px 19px 38px, rgba(0, 0, 0, 0.22) 0px 15px 12px",
+            boxShadow:
+                "rgba(0, 0, 0, 0.3) 0px 19px 38px, rgba(0, 0, 0, 0.22) 0px 15px 12px",
```
Comment (alicolver) [inline src/leaderboard/PointCard.tsx]: do we have line lengths? think it's better before

File: src/leaderboard/PointCard.tsx [hunk 4]
```diff
@@ -1,57 +1,109 @@
...
 
+    useEffect(() => {
+        setPointsDataIsLoading(true)
+        fetch(resolveEndpoint("points"), {
+            method: "GET",
+            headers: {
+                Authorization: getJWT(),
+            },
+        })
+            .then((res) => {
```
Comment (alicolver) [inline src/leaderboard/PointCard.tsx]: i think this can go on previous line

**Review by alicolver — APPROVED** [review]

File: src/homepage/Homepage.tsx (hunk 1 above)
Comment (alicolver) [reply src/homepage/Homepage.tsx:29]: i think it's probably cleaner if you have loads of fonts or smthn

**Pull request merged**